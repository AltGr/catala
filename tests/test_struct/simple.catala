@Article@

/*
new struct S:
  data x type int
  data y type int 

new scope A:
  param s type S 
  param z type int 

scope A:
  def s := S {
    -- x : 1 
    -- y : 2
  }
  def z := s.x + s.y
*/