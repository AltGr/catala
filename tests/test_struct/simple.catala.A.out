[RESULT] s -> ("x": 1,"y": 2)
[RESULT] z -> 3
